src/LinuxTracing/ThreadStateManagerTest.cpp

```diff
@@ -38,6 +38,7 @@ TEST(ThreadStateManager, OneThread) {
   EXPECT_EQ(slice->wakeup_reason(), orbit_grpc_protos::ThreadStateSlice::kNotApplicable);
   EXPECT_EQ(slice->wakeup_pid(), kWakeupPidTidWhenWakeupReasonNotApplicable);
   EXPECT_EQ(slice->wakeup_tid(), kWakeupPidTidWhenWakeupReasonNotApplicable);
+  EXPECT_EQ(slice->callstack_status(), orbit_grpc_protos::ThreadStateSlice::kNoCallstack);
```

Comment:
Even if it doesn't matter, I think we should also check that the callstack id is set to 0, just like `wakeup_pid` and `wakeup_tid` are checked.

Reply:
Done.